src/components/breadcrumbs/index.md.njk

```diff
@@ -70,11 +76,39 @@ There are two ways to use the Breadcrumbs component. You can use HTML or, if you
 HTML
 
 ```html
+<div class="govuk-c-breadcrumb">
```

Comment:
This feels like it's a duplication of the content in the tabs of the example? Is that intentional?

Reply:
These were used to illustrate how large code blocks are being styled. Will be removed in #84 when updating the content for each component. I can remove them from this PR if they're not useful for testing.